src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -205,7 +205,19 @@ private function parseDefinition(\DOMElement $service, string $file, array $defa
             }
 
             if ($deprecated = $this->getChildren($service, 'deprecated')) {
-                $alias->setDeprecated(true, $deprecated[0]->nodeValue ?: null);
+                $message = $deprecated[0]->nodeValue ?: '';
+                $package = $deprecated[0]->getAttribute('package') ?: '';
+                $version = $deprecated[0]->getAttribute('version') ?: '';
+
+                if ('' === $package) {
```

Comment:
actually the attribute can be set and be empty - this should be
`!$deprecated[0]->hasAttribute('package')` instead (same below)

Reply:
done